controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -490,14 +490,6 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
         }
     }
 
-    std::string manufacturer =
```

Comment:
I think it is beneficial to still have the manufacturer at least printed out here, and also add the manufacturer to the node structure in the DB so it will be seen in the network map.

Reply:
There is currently nothing in the DB for this information.

If we consider adding it, then maybe we should first think about refactoring the database in general...

Reply:
(nitpick) perhaps we can just print the manufacturer here so we'll have it in the log?